docs/elements.md

```diff
@@ -288,7 +288,7 @@ and external grading system.
 
 #### Sample element
 
-![](elements/pl-file-editor.png)
+![Screenshot of the file editor element](elements/pl-file-editor.png)
```

Comment:
Let's be consistent. Sometimes you use the element name, sometimes you don't. I'd prefer to unconditionally use the element name.